bundle.Dockerfile.cross

```diff
@@ -0,0 +1,20 @@
+FROM --platform=${BUILDPLATFORM} scratch
```

Comment:
IMO. Not sure we actually need to build catalogs as part of the release because catalogs aren't really pulled/used directly outside of something like operatorhub which builds the catalog themselves.

Reply:
👍 removed